lib/pbench/test/unit/server/test_file_tree.py

```diff
@@ -358,29 +365,22 @@ def mock_rmtree(path: Path, ignore_errors=False):
             assert ignore_errors
             assert path == incoming / f"{tarball_name}.unpack"
 
-        def mock_run(args, **kwargs):
-            if args[0] == "tar":
-                return subprocess.CompletedProcess(
-                    args, returncode=0, stdout="Successfully Unpacked!", stderr=None
-                )
-            else:
-                assert args[0] == "find"
-                raise subprocess.TimeoutExpired("find", 43)
+        @staticmethod
+        def mock_run(command, dir_path, exception, dir_p):
+            verb = "find"
+            if command.startswith(verb):
+                raise subprocess.TimeoutExpired(verb, 43)
 
```

Comment:
If `command` does not start with `"find"`, then we should confirm that it starts with `"tar"`:
```
        def mock_run(command, dir_path, exception, dir_p):
            verb = "find"
            if command.startswith(verb):
                raise subprocess.TimeoutExpired(verb, 43)
            else:
                assert command.startswith("tar")
```